tools/test-backend

```diff
@@ -66,6 +67,56 @@ if __name__ == "__main__":
                       help="Show which tests are slowest.")
 
     (options, args) = parser.parse_args()
+
+    thedir = 'zerver/tests/'
+
+    for suite in args:
+        args[args.index(suite)]=suite.replace("/", ".")
+
+    for suite in args:
+        flag=0
```

Comment:
Also, I'd consider using a more clear name, like `found` to make it more clear on reading the code what this flag is for.